snapcraft/tests/commands/test_collaborate.py

```diff
@@ -0,0 +1,92 @@
+# -*- Mode:Python; indent-tabs-mode:nil; tab-width:4 -*-
+#
+# Copyright (C) 2016 Canonical Ltd
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License version 3 as
+# published by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+import logging
+
+from unittest import mock
+
+import fixtures
+
+from snapcraft import (
+    storeapi,
+    tests
+)
+
+from snapcraft._store import collaborate
+
+
+class CollaborateTestCase(tests.TestCase):
+
+    def setUp(self):
+        super().setUp()
+        self.fake_logger = fixtures.FakeLogger(level=logging.INFO)
+        self.useFixture(self.fake_logger)
+        self.fake_store = tests.fixture_setup.FakeStore()
+        self.useFixture(self.fake_store)
+        self.client = storeapi.StoreClient()
+        patcher = mock.patch('snapcraft._store.Popen')
+        self.popen_mock = patcher.start()
+        rv_mock = mock.Mock()
```

Comment:
Honestly speaking I copied and pasted this part from `test_validate.py` for the tests to work. (without not being able to get pdb working, it was hard to figureout what that was.)  Now that you asked, did a bit of research and that is a process_mock afaiu. Renamed it accordingly. Thanks